.gitignore

```diff
@@ -16,6 +16,8 @@
 /platform/android/build
 /platform/android/MapboxGLAndroidSDK/src/main/assets/sdk_versions/com.mapbox.mapboxsdk
 *.code-workspace
+
```

Comment:
Does Mapbox.playground allow for adding files to Git ignore?

That is, is it possible to remove the Playground binary build files by adding the following?

```
platform/ios/platform/ios/Mapbox.playground/build/
```

Reply:
Should be fine now